docs/src/main/sphinx/connector/snowflake.rst

```diff
@@ -0,0 +1,88 @@
+===================
+Snowflake connector
+===================
+
+The Snowflake connector allows querying and creating tables in an
+external Snowflake account. This can be used to join data between
+different systems like Snowflake and Hive, or between two different
+Snowflake accounts.
+
+Configuration
+-------------
+
+To configure the Snowflake connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``snowflake.properties``, to
```

Comment:
We use `example.properties` as the template. See other connector docs.

Reply:
Updated

Reply:
> Updated

Did you forget to push the changes?

Reply:
> I'll let you know when it has done.

Please avoid leaving "Updated" comments until this PR is updated. The comment is confusing and not actionable.